SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialTypebindingManager.cpp

```diff
@@ -15,6 +15,8 @@
 #include "EngineClasses/SpatialPackageMapClient.h"
 #include "Utils/RepLayoutUtils.h"
 
+DEFINE_LOG_CATEGORY(LogTypebindingManager);
```

Comment:
Can this be changed to `LogSpatialTypebindingManager`

Reply:
Done